### PR TITLE
Bump SLES to SP5 with docker 19

### PR DESCRIPTION
**Pull request by obierlaire**
Closes https://github.com/elastic/cloud/issues/73576

This is bumping SLES_12 to **SP5** and **docker 19**

### CODE REVIEW

**Comment by obierlaire** [comment]
According to the [validation matrix pipeline](https://cloud-ci.elastic.co/blue/organizations/jenkins/jjbb%2Belastic%2Bcloud%2Bece-support-matrix-validation/detail/jjbb%20elastic%20cloud%20ece-support-matrix-validation/217/pipeline/), this is working. 
image
NB: the red steps are already known issues regarding centos/rhel7, totally unrelated to this PR

File: tasks/base/SLES-12/install_docker.yml [hunk 1]
```diff
@@ -12,35 +12,6 @@
   delay: 30
   until: remove_packages is success
 
-- name: Add docker repository
```
Comment (obierlaire) [inline tasks/base/SLES-12/install_docker.yml:15]: SP 5 comes already with a suitable repo for docker 19, so no need to install an other one

**Comment by Kushmaro** [comment]
@obierlaire we bump the ansible installer, but I'm not sure if @vogxn 's team ever ran a test with SLES 12  +  Docker 19
We should work towards having this as part of our OS testing pipeline.

**Comment by obierlaire** [comment]
> @obierlaire we bump the ansible installer, but I'm not sure if @vogxn 's team ever ran a test with SLES 12 + Docker 19
> We should work towards having this as part of our OS testing pipeline.

I think this is covered by https://github.com/elastic/cloud/pull/73954 where we bump our matrix validation pipeline to test this. @vogxn do you think this is enough?

**Review by vaubarth — APPROVED** [review]

**Comment by vaubarth** [comment]
I think having it in the ECE support matrix job is exactly the kind of validation we want for this

**Review by Kushmaro — APPROVED** [review]

**Pull request merged**